source/tests/system/nitclk_driver_api_tests.cpp

```diff
@@ -1,8 +1,7 @@
 #include <gtest/gtest.h>
 
-#include "niscope/niscope_library.h"
+#include "device_server.h"
```

Comment:
@doshirohan , I could not find changes for tclk session tests

Reply:
We do not have session tests for tclk.

Reply:
Oh right. Sorry had forgot